scripts/ci_test_mqtt.sh

```diff
@@ -16,28 +16,32 @@ DEMO_OPTIONS="-i $IOT_IDENTIFIER"
 if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then
     if [ "$TRAVIS_OS_NAME" = "linux" ]; then
         # Set the flags and options for a local Mosquitto broker on Linux.
-        CMAKE_FLAGS+=" -DIOT_TEST_MQTT_MOSQUITTO=1 -DIOT_TEST_SERVER=\\\"localhost\\\""
+        CMAKE_CREDENTIAL_FLAGS+=" -DIOT_TEST_MQTT_MOSQUITTO=1 -DIOT_TEST_SERVER=\\\"localhost\\\""
         DEMO_OPTIONS+=" -n -u -h localhost -p 1883"
     fi
 else
     # Set credentials for AWS IoT.
-    CMAKE_FLAGS+=" $AWS_IOT_CREDENTIAL_DEFINES"
+    CMAKE_CREDENTIAL_FLAGS="$AWS_IOT_CREDENTIAL_DEFINES"
```

Comment:
Made this string concatenation change as Travis CI builds (on my account) unexpolicably give the following error:
`../scripts/ci_test_mqtt.sh: 24: ../scripts/ci_test_mqtt.sh: CMAKE_FLAGS+= -DIOT_TEST_SERVER="\"[secure]\"" -DIOT_TEST_PORT=443 -DIOT_TEST_ROOT_CA="\"../credentials/AmazonRootCA1.pem\"" -DIOT_TEST_CLIENT_CERT="\"../credentials/clientCert.pem\"" -DIOT_TEST_PRIVATE_KEY="\"../credentials/privateKey.pem\"": not found`